src/MassTransit/Configuration/ProcessingTimeoutConfiguratorExtensions.cs

```diff
@@ -0,0 +1,88 @@
+﻿namespace MassTransit
+{
+    using System;
+    using ConsumeConfigurators;
+    using GreenPipes;
+    using PipeConfigurators;
+    using Saga;
+
+
+    public static class ProcessingTimeoutConfiguratorExtensions
+    {
+        /// <summary>
+        /// TODO
+        /// </summary>
+        /// <param name="configurator">The pipe configurator</param>
+        /// <param name="configure">Configure processing timeout</param>
+        public static void UseProcessingTimeout<T>(this IPipeConfigurator<ConsumeContext<T>> configurator, Action<IProcessingTimeoutConfigurator> configure = default)

```

Comment:
I'm going to say let's just make it `UseTimeout()`, to eliminate the Processing from, well, everywhere. Shorter, and gets the point across.

Reply:
Yeah, couldn't agree more. Fixed.